### PR TITLE
Alc 92

### PR DESCRIPTION
**Pull request by pygeek**
Please ensure have read our (contribution guidelines](CONTRIBUTING.md) and your
pull request adheres to the following guidelines:

- [x] Branch name corresponds to Linear ticket number.
- [x] Added sufficient integration and unit tests.
- [x] Addressed a single concern in the least number of changed lines as
      possible.
- [x] Only fixed/added the functionality in question **OR** address wide-spread
      whitespace/style issues, not both.

Thanks!

### CODE REVIEW

File: src/components/PoolPriceChart/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Index from "index";
+import { MockedProvider } from "@apollo/client/testing";
+
+test("renders PoolPriceChart component", () => {
```
Comment (ledes) [inline src/components/PoolPriceChart/index.test.tsx:6]: check  this warning

File: src/components/PoolPriceChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      poolTotalSwapVolume: r.poolTotalSwapVolume,
+      priceUsd: r.priceUsd,
+      timestamp: DateTime.fromMillis(r.timestamp).toFormat("HH:mm"),
+    };
+  });
+};
+
+export default function PoolPriceChart() {
+  const data: any = getPoolMetrics();
+  console.log(data);
```
Comment (ledes) [inline src/components/PoolPriceChart/index.tsx:82]: we can probably have a loading state for this component but we can add it later.
Thoughts on a loading state?

File: src/components/PoolPriceChart/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Index from "index";
+import { MockedProvider } from "@apollo/client/testing";
+
+test("renders PoolPriceChart component", () => {});
```
Comment (hosainnet) [inline src/components/PoolPriceChart/index.test.tsx:6]: Nothing here?

File: src/components/PoolPriceChart/index.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      poolLiquidity: r.poolLiquidity,
+      poolTotalSwapVolume: r.poolTotalSwapVolume,
+      priceUsd: r.priceUsd,
+      timestamp: DateTime.fromMillis(r.timestamp).toFormat("HH:mm"),
+    };
+  });
+};
+
+export default function PoolPriceChart() {
+  const data: any = getPoolMetrics();
```
Comment (hosainnet) [inline src/components/PoolPriceChart/index.tsx:81]: There is a type of this data in `types/PoolMetrics.ts`

**Comment by ledes** [comment]
I think this chart could be similar to what we want. The only thing I don't know if we will be able to do is the popover or tooltip with the price
https://recharts.org/en-US/examples/ComposedChartWithAxisLabels

**Comment by pygeek** [comment]
@ledes @hosainnet the code in PoolPriceChart is just for reference, it can all be deleted once @ledes does his ticket.

**Review by hosainnet — APPROVED** [review]

**Review by ledes — APPROVED** [review]

**Pull request merged**